app/src/main/java/com/internship/move/MyApp.kt

```diff
@@ -0,0 +1,29 @@
+package com.internship.move
+
+import android.app.Application
+import com.internship.move.di.repositories
+import com.internship.move.di.storage
+import com.internship.move.di.viewModels
+import org.koin.android.ext.koin.androidContext
+import org.koin.core.context.startKoin
+
+class MyApp : Application() {
```

Comment:
I suggest naming this to something more appropriate for the project. Maybe `MoveApp` or similar